### PR TITLE
Add Study Variations

### PR DESCRIPTION
**Pull request by bentoncase**
@gregglind how do these variations look? 
r?

### CODE REVIEW

File: lib/variations.js [hunk 1]
```diff
@@ -8,6 +9,11 @@ const variations = {
   'default'() {
     Rec.start();
   },
+  'autoOpenPanel'() {
+    simplePrefs.prefs.autoOpenPanel = true;
+    Rec.start();
+  },
+  'contol'() {},
```
Comment (gregglind) [inline lib/variations.js]: this is a typo :)  I might suggest 'observe-only'.

Rename 'default' to "passive-panel" maybe?  These three names are doing to show up in the all the analysis and should self document

**Comment by gregglind** [comment]
F+ on design, R- because a few nits.

**Comment by bentoncase** [comment]
Ok, I made the changes. r?

**Comment by gregglind** [comment]
R+ now.  Thanks!

**Pull request merged**